frappe/utils/background_jobs.py

```diff
@@ -13,15 +13,19 @@
 import pymysql
 from pymysql.constants import ER
 
-default_timeout = 300
-queue_timeout = {
-	'background': 2500,
-	'long': 1500,
-	'default': 300,
-	'short': 300
-}
-
 redis_connection = None
+queue_timeout  = {}
```

Comment:
This could be done. But managing the config becomes difficult.
Adding a new default queue would need changes in supervisor config builder. That makes two different places to update for a single config change.

Having them in the config will be required for build the supervisor config. Supervisor config update is done here https://github.com/frappe/bench/pull/740

Reply:
frappe/bench#739 will handle this issue. This is alright.

Reply:
@sunnyakaxd : patch is already added in this https://github.com/frappe/bench/pull/739/files